products/netbsd.md

```diff
@@ -0,0 +1,69 @@
+---
+permalink: /netbsd
+title: NetBSD
+category: os
+versionCommand: uname -r
+activeSupportColumn: false
+releasePolicyLink: https://www.netbsd.org/releases/
+releaseDateColumn: true
+releaseColumn: false
+changelogTemplate: https://www.netbsd.org/releases/formal___RELEASE_CYCLE__/__LATEST__.html
```

Comment:
i think this formal_ part should be formal-

Reply:
@usta Thanks again for your quick responses and putting up with all of my bugs and typos.  See the latest update to the [NetBSD branch](https://github.com/alexkoster/endoflife.date/tree/netbsd) to check if its working on your end.